.gitmodules

```diff
@@ -59,3 +59,6 @@
 [submodule "third_party/cpuinfo"]
 	path = third_party/cpuinfo
 	url = https://github.com/pytorch/cpuinfo.git
+[submodule "third_party/llvm-bazel"]
+	path = third_party/llvm-bazel
+	url = https://github.com/GMNGeoffrey/llvm-bazel.git
```

Comment:
Do you still want this to point to your fork?

Reply:
OOoh no. I thought I fixed that!